src/testHelpers/createMockConfig.js

```diff
@@ -25,6 +25,16 @@ const createMockConfig = ({ clientSide } = {}) => {
       databaseURL: 'https://my-example-app.firebaseio.com',
       projectId: 'my-example-app-id',
     },
+    redirectConfig: {
+      authenticatedUser: {
+        destination: '/app',
+        permanenent: false,
+      },
+      unauthenticatedUser: {
+        destination: '/app',
```

Comment:
This would probably be `/` or `/login` right? To be different than authenticated?

Reply:
Yeah, this is correct. The value here was just a placeholder for the moment.